config/crd/bases/anywhere.eks.amazonaws.com_controlplaneupgrades.yaml

```diff
@@ -74,6 +74,11 @@ spec:
                     description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
                     type: string
                 type: object
+              controlPlaneSpec:
```

Comment:
Is this the entire kcp object? Also, I am forgetting why we need controlPlane and controlPlaneSpec separately and if we can somehow explain that in both of the descriptions.

Reply:
done